plugin/evm/vm_test.go

```diff
@@ -3016,6 +2964,21 @@ func TestSkipChainConfigCheckCompatible(t *testing.T) {
 	require.NoError(t, reinitVM.Shutdown(context.Background()))
 }
 
+func TestMandatoryUpgrades(t *testing.T) {
+	// use modified genesis to set SubnetEVM timestamp to 1
+	genesisJsonModifiedSubnetEVM := strings.Replace(genesisJSONSubnetEVM, "\"subnetEVMTimestamp\":0", "\"subnetEVMTimestamp\":1", 1)
+	_, vm, _, _ := GenesisVM(t, true, genesisJsonModifiedSubnetEVM, "", "")
+
+	defer func() {
+		if err := vm.Shutdown(context.Background()); err != nil {
+			t.Fatal(err)
```

Comment:
nit: can use require here.

Reply:
I used this to keep the convention with other tests, but yea I think we can change it. 

We should definitely modify these tests to use require. Created an issue https://github.com/ava-labs/subnet-evm/issues/701